### PR TITLE
DOCS-1892-CE-3.18-2

### PR DESCRIPTION
**Pull request by bmckercher123**
Product Version(s):
CE 3.18-2

Issue:

Link to docs preview:

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -371,7 +371,7 @@ const config = {
         path: 'calico-enterprise',
         routeBasePath: 'calico-enterprise',
         editCurrentVersion: true,
-        onlyIncludeVersions: ['3.18','3.17','3.16','3.15'],
+        onlyIncludeVersions: ['3.18-2','3.18','3.17','3.16','3.15'],
```
Comment (ctauchen) [inline docusaurus.config.js]:
```suggestion
        onlyIncludeVersions: ['3.18-2','3.17','3.16','3.15'],
```

File: calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx [hunk 2]
```diff
@@ -112,24 +115,30 @@ For more information, see [link](https://docs.tigera.io).
 
 ### Calico Enterprise 3.18.0 (early preview)
 
-1 September 2023
+01 September 2023
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx]: We should change all dates to Month DD, YYYY format. See https://developers.google.com/style/dates-times#expressing-dates

An oversight when I was getting the template together.

File: calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx [hunk 3]
```diff
@@ -48,62 +84,29 @@ We've added new options for Helm installs including: set the number of replicas
...
-This option augments support for the Azure CNI with no overlay (where a VNET IP address is assigned to every pod).
-
-To review Azure CNI options, see [Microsoft Azure Kubernetes Service (AKS)](../getting-started/install-on-clusters/aks).
-
-<!-- We can break this up by component. H3 subheads or additions to titles -->
-
 ### Deprecated and removed features
 
-* The anomaly detection feature has been removed in this release.
+* The anomaly detection feature is removed in this release
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx]: End punctuation for complete sentences in lists: https://developers.google.com/style/lists#types-of-lists

File: calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx [hunk 4]
```diff
@@ -48,62 +84,29 @@ We've added new options for Helm installs including: set the number of replicas
...
+* [Manual install for Windows](../getting-started/install-on-clusters/windows-calico/manual-install/) will be removed in a future release. Starting in this release, the [standard installation is operator-based](../getting-started/install-on-clusters/windows-calico/operator).
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx]:
```suggestion
* [Manual install for Windows](../getting-started/install-on-clusters/windows-calico/manual-install.mdx) will be removed in a future release. Starting in this release, the [standard installation is operator-based](../getting-started/install-on-clusters/windows-calico/operator.mdx).
```
I'm wondering whether we should swap "will be removed" with "deprecated". What do you think?

File: calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx [hunk 5]
```diff
@@ -4,22 +4,58 @@ description: What's new, and why features provide value for upgrading.
 
 # Calico Enterprise 3.18 release notes
 
-:::info early preview release
+Learn about the new features, bug fixes, and other updates in this release of {{prodname}}. For a list of component versions that are included this release, see [Component versions](../reference/component-versions).
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx]:
```suggestion
Learn about the new features, bug fixes, and other updates in this release of {{prodname}}. For a list of component versions that are included this release, see [Component versions](../reference/component-versions.mdx).
```
Good practice for all markdown links. Helps with versioning, particularly if files get moved around.

File: calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx [hunk 6]
```diff
@@ -4,22 +4,58 @@ description: What's new, and why features provide value for upgrading.
...
+## New features and enhancements
 
-:::
+### VXLAN support for cluster mesh and federation
 
-Learn about the new features, bug fixes, and other updates in this release of {{prodname}}.
+We've expanded our support of cluster mesh to clusters using VXLAN for networking. Cluster mesh can be used to federate services and endpoints to authorize cross-cluster communication with {{prodname}} network policies.
 
-## New features and enhancements
+For more information, see [Configure federated endpoint identity and services](../multicluster/federation/kubeconfig).
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx]:
```suggestion
For more information, see [Configure federated endpoint identity and services](../multicluster/federation/kubeconfig.mdx).
```
Same throughout.

File: calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx [hunk 7]
```diff
@@ -4,22 +4,64 @@ description: What's new, and why features provide value for upgrading.
...
+
+### Helm-based install for multi-cluster management deployments
+
+We've added the support for Helm for installing and setting up deployments that use the multi-cluster management feature. With Helm, DevOps teams can automate these complex deployments more easily.
+
+For more information, see [Helm for multi-cluster management](../multicluster/set-up-multi-cluster-management/standard-install/).
+
+### HPC-based Windows install and upgrade
+
+We've added support for Operator-based install and upgrade for Windows nodes using HPC (HostProcess/Privileged containers). {{prodname}} can now bootstrap Windows worker nodes just like Linux worker nodes. Platform engineers can scale Windows worker nodes with an auto-scaling node pool without bootstrapping each node manually, and upgrade {{prodname}} deployments without manually upgrading nodes one at a time.
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx]:
```suggestion
We've added support for operator-based install and upgrade for Windows nodes using HPC (HostProcess/Privileged containers). {{prodname}} can now bootstrap Windows worker nodes just like Linux worker nodes. Platform engineers can scale Windows worker nodes with an auto-scaling node pool without bootstrapping each node manually, and upgrade {{prodname}} deployments without manually upgrading nodes one at a time.
```

File: calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx [hunk 8]
```diff
@@ -4,22 +4,64 @@ description: What's new, and why features provide value for upgrading.
...
+
+For more information, see [the {{prodnameWindows}} Operator guide](../getting-started/install-on-clusters/windows-calico/operator).
+
+### Support for Azure CNI with overlay networking for AKS
+
+We now officially support AKS clusters that are using overlay networking. This option is useful if you've exhausted your IP addresses. This option augments existing support for Azure CNI with no overlay (where a VNET IP address is assigned to every pod).
+
+To review Azure CNI options, see [Microsoft Azure Kubernetes Service (AKS)](../getting-started/install-on-clusters/aks).
+
+### Support for Kubernetes v2.18
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx]:
```suggestion
### Support for Kubernetes 1.28
```
Right?

**Review by ctauchen — COMMENTED** [review]
Looking good! A few small suggestions below.

We might consider adding a metadata line: 

`title: Release notes`

That should get the shorter title in the sidebars while leaving the long title on the page.

File: calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx (hunk 4 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx]: That's fine with me. Fixed.

File: calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx (hunk 2 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx]: Fixed.

File: calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx (hunk 3 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx]: Fixed.

File: calico-enterprise_versioned_docs/version-3.18-2/releases.json [hunk 9]
```diff
@@ -1,9 +1,9 @@
 [
   {
-    "title": "master",
+    "title": "v3.18.0-2.0",
     "tigera-operator": {
       "image": "tigera/operator",
-      "version": "master",
+      "version": "v1.31.2",
```
Comment (sabags) [inline calico-enterprise_versioned_docs/version-3.18-2/releases.json]:
```suggestion
      "version": "v1.32.2",
```

File: calico-enterprise_versioned_docs/version-3.18-2/releases.json [hunk 10]
```diff
@@ -13,237 +13,789 @@
...
       },
       "honeypod-controller": {
         "image": "tigera/honeypod-controller",
-        "version": "master"
+        "version": "v3.18.0-2.0"
       },
       "key-cert-provisioner": {
         "image": "tigera/key-cert-provisioner",
-        "version": "master",
+        "version": "v1.1.13",
```
Comment (danudey) [inline calico-enterprise_versioned_docs/version-3.18-2/releases.json]: v1.1.15

File: calico-enterprise_versioned_docs/version-3.18-2/releases.json [hunk 11]
```diff
@@ -13,237 +13,789 @@
...
+        "version": "7.17.13"
       },
       "kibana": {
         "image": "tigera/kibana",
-        "version": "master"
+        "version": "v3.18.0-2.0"
       },
       "eck-elasticsearch": {
-        "version": "7.17.9"
+        "version": "7.17.13"
```
Comment (danudey) [inline calico-enterprise_versioned_docs/version-3.18-2/releases.json]: 7.17.14

File: calico-enterprise_versioned_docs/version-3.18-2/releases.json [hunk 12]
```diff
@@ -13,237 +13,789 @@
...
+        "version": "v3.18.0-2.0"
       },
       "es-proxy": {
         "image": "tigera/es-proxy",
-        "version": "master"
+        "version": "v3.18.0-2.0"
       },
       "eck-kibana": {
-        "version": "7.17.9"
+        "version": "7.17.13"
```
Comment (danudey) [inline calico-enterprise_versioned_docs/version-3.18-2/releases.json]: 7.17.14

File: calico-enterprise_versioned_docs/version-3.18-2/releases.json [hunk 13]
```diff
@@ -13,237 +13,789 @@
...
+        "version": "v3.18.0-2.0"
       },
       "prometheus": {
         "image": "tigera/prometheus",
-        "version": "master"
+        "version": "v3.18.0-2.0"
       },
       "coreos-prometheus": {
-        "version": "v2.42.0"
+        "version": "v2.45.0"
```
Comment (danudey) [inline calico-enterprise_versioned_docs/version-3.18-2/releases.json]: v2.47.0

**Review by danudey — COMMENTED** [review]
Just a few version updates required.

File: calico-enterprise_versioned_docs/version-3.18-2/releases.json (hunk 9 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.18-2/releases.json]: Fixed.

File: calico-enterprise_versioned_docs/version-3.18-2/releases.json (hunk 10 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.18-2/releases.json]: Fixed.

File: calico-enterprise_versioned_docs/version-3.18-2/releases.json (hunk 11 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.18-2/releases.json]: Fixed.

File: calico-enterprise_versioned_docs/version-3.18-2/releases.json (hunk 12 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.18-2/releases.json]: Fixed.

File: calico-enterprise_versioned_docs/version-3.18-2/releases.json (hunk 13 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.18-2/releases.json]: Fixed.

**Pull request closed without merging**